### PR TITLE
Make compatible with java 14

### PR DESCRIPTION
**Pull request by hadisfr**
### :pencil: Description

Make it possible to build project with java 14

### CODE REVIEW

**Comment by lasanthak** [comment]
Thanks @hadisfr for your PR. I will have a look. We are working on migrating the build system from Travis to Git Workflow

File: jarviz-lib/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     <properties>
```
Comment (lasanthak) [inline jarviz-lib/pom.xml:30]: @hadisfr - I just finished migrating to GH Action and as part of it I upgraded libs and fixed some warnings. So I just realized you also did the same here. We have latest ASM version 9.1 now with the release of 0.1.4.

File: jarviz-lib/pom.xml [hunk 2]
```diff
@@ -231,6 +231,13 @@
 
     <build>
         <plugins>
+            <plugin>
```
Comment (lasanthak) [inline jarviz-lib/pom.xml:234]: Without realizing, I've even fixed this
https://github.com/ExpediaGroup/jarviz/blob/master/pom.xml#L128-L134

File: jarviz-lib/src/main/java/com/vrbo/jarviz/AnalyzeCommand.java [hunk 3]
```diff
@@ -164,7 +165,9 @@ private static void reportError(final Exception e) {
             System.err.println("Analyze failed");
             e.printStackTrace();
         } else {
-            System.err.println("Analyze failed: " + e.getMessage());
```
Comment (lasanthak) [inline jarviz-lib/src/main/java/com/vrbo/jarviz/AnalyzeCommand.java:167]: Will bring this in

File: jarviz-lib/pom.xml (hunk 1 above)
Comment (hadisfr) [reply jarviz-lib/pom.xml]: So this is already done. :D

File: jarviz-lib/pom.xml (hunk 2 above)
Comment (hadisfr) [reply jarviz-lib/pom.xml:234]: And this is already done, too.

File: jarviz-lib/src/main/java/com/vrbo/jarviz/AnalyzeCommand.java (hunk 3 above)
Comment (hadisfr) [reply jarviz-lib/src/main/java/com/vrbo/jarviz/AnalyzeCommand.java:167]: I merge your new master into this branch to resolve merge conflicts.

**Pull request merged**